NTO/Automation/entities/ActionHandlerCapability.ttl

```diff
@@ -0,0 +1,29 @@
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix ogit.Automation:        <http://www.purl.org/ogit/Automation/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Automation:ActionHandlerCapability
+    a rdfs:Class;
+    rdfs:subClassOf ogit:Entity;
+    rdfs:label "ActionHandlerCapability";
+    dcterms:description """Configuration node to dynamically connect Knowledge Pools or Rankings to a HIRO installation.""";
```

Comment:
Description should be correct, this is a copy&paste version from configuration node